Assets/Scripts/HoldObject.cs

```diff
@@ -70,6 +71,7 @@ private void CarryObject()
     IEnumerator Release()
     {
         setPlayerControl(false);
+        GetComponent<PlayerMovement>().isKoalaPickedUp = false;
```

Comment:
please use `IsHolding` property in this script from within PlayerMovement. It's meant to work that way

Reply:
ah, missed it